core/amber/src/main/python/core/models/marker.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+
+from proto.edu.uci.ics.amber.engine.common import LinkIdentity
+
+
+@dataclass
+class Marker:
```

Comment:
Can you add some comments explaining what's a `Marker`? I assume it's an interface that extended by "data messages" that are not data, but serves control purposes.

Reply:
done